spec/dummy/config/environments/test.rb

```diff
@@ -16,7 +16,7 @@
   # Configure public file server for tests with Cache-Control for performance.
   config.public_file_server.enabled = true
   config.public_file_server.headers = {
-    'Cache-Control' => "public, max-age=#{1.hour.to_i}"
+    'Cache-Control' => "public, max-age=#{3600}"
```

Comment:
This change was enough to get the specs running locally for me. Happy to revert, but wanted to include in case this feels like a worthwhile change.